travis/travis_run_tests

```diff
@@ -53,9 +53,15 @@ if __name__ == '__main__':
     tests_enabled = os.environ.get('TESTS') == '1'
     tests_unspecified = os.environ.get('TESTS') is None
     transifex_enabled = os.environ.get('TRANSIFEX') == '1'
+    is_oca_project = os.environ.get('TRAVIS_REPO_SLUG', '').startswith('OCA/')
+    is_oca_transifex_user = os.environ.get('TRANSIFEX_USER') == \
+        'transbot@odoo-community.org'
 
     # TRAVIS_PULL_REQUEST contains the pull request number or 'false'
     is_pull_request = os.environ.get('TRAVIS_PULL_REQUEST') != 'false'
+    # Avoid false red from forks using OCA transifex user
+    is_valid_transifex = not is_pull_request and \
+        is_oca_project == is_oca_transifex_user
```

Comment:
I see, your solution is correct but the code obfuscates the reasoning behind it. We should improve it.
IMO we shouldn't worry about the 2nd case in the table.
This means that we should build Transifex when `is_oca_project or not is_oca_transifex_user`:

```
is_valid_transifex = not is_pull_request and \
    (is_oca_project or not is_oca_transifex_user)
```